Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Vapes/vape.yml

```diff
@@ -0,0 +1,17 @@
+- type: Tag
```

Comment:
put this tag with the rest of the tags in `tags.yml`. On second thought, I'm not even sure you really need a tag? It seems to be unused besides being assigned here.